### ISSUE TITLE
SS 3.2.0-rc1 - Wrong interface implementation of ImagickBackend::imageAvailable() 

### ISSUE THREAD

**Issue by hpeide** [silverstripe/silverstripe-framework#4596]
There's a interface implementation error with ImagickBackend::imageAvailable() method.

Somthing like this would fix it.

framework/filesystem/ImagickBackend.php line 98 -106
```php
/**
 * @todo Implement memory checking for Imagick? See {@link GD}
 *
 * @param string $filename
 * @return boolean
 */
public function imageAvailable($filename, $manipulation) {
    return true;
}
```

**Comment by kinglozzer** [comment]
@hpeide Nice find, I think I’m to blame for that! Fancy submitting a PR against the 3.2 branch? :)

**Comment by kinglozzer** [comment]
It might be worth us adding Imagick to our Travis builds - installation doesn’t seem to slow builds down much: https://github.com/kinglozzer/sapphire/commit/f42aba62bbfd877bcbc285e9d074f9d675af5cc8. Builds with Imagick added are currently failing because of this issue of course: https://travis-ci.org/kinglozzer/sapphire/jobs/79669387

**Comment by tractorcow** [comment]
Fix at https://github.com/silverstripe/silverstripe-framework/pull/4653

**Comment by kinglozzer** [comment]
Closed via 7ec24a13031b3daf5579d6b3e0a89f0119bb2ac0
